romancal/tweakreg/tests/test_tweakreg.py

```diff
@@ -727,6 +638,8 @@ def test_tweakreg_common_name_raises_error_on_invalid_input(filename_list):
 )
 def test_tweakreg_save_valid_abs_refcat(tmp_path, abs_refcat, request):
     """Test that TweakReg saves the catalog used for absolute astrometry."""
+    os.chdir(tmp_path)
```

Comment:
Is this needed? Will the step respect `output_dir` for writing the catalog?

Reply:
Not needed anymore since the step will respect `output_dir`.